.github/scripts/check_reference_conf.py

```
@@ -0,0 +1,319 @@
+    for full_path, _depth, is_leaf in keys:
+        if not is_leaf:
+            continue
+        if "[]" in full_path:
+            continue
+        seg = full_path.split(".")[-1]
+        if not _is_port_segment(seg):
+            continue
+        try:
+            raw = tree.get(full_path)
+        except Exception:
+            continue
```

Ruff (0.15.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)